### DISCUSSION TITLE
middleware/minio/user-guide/update/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1908] [Announcements]
# middleware/minio/user-guide/update/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/middleware/minio/user-guide/update/

**Comment by liyuerich** [comment]
3.修改规格配置，然后点击下一步。
仅支持修改：CPU 配额和内存配额 
不可修改：版本、部署模式、存储类、容量

这个图比较旧了，现在还支持修改磁盘数

**Reply by windsonsea** [reply]
Fixed in PR #5300
